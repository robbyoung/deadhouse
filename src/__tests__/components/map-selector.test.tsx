import React from 'react';
import {BrowserRouter as Router, Switch, Route} from 'react-router-dom';
import renderer from 'react-test-renderer';
import MapSelector from '../../components/map-selector';

describe('MapSelector component', () => {
    it('Can render a list of map buttons', () => {
        const component = renderer.create(
            <Router>
                <MapSelector />
                <Switch>
                    <Route path="/">
                        <div />
                    </Route>
                </Switch>
            </Router>,
        );

        expect(component).toMatchSnapshot();
    });
});
