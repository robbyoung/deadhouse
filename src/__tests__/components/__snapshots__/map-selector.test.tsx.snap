// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapSelector component Can render a list of map buttons 1`] = `
Array [
  <div
    className="map-selection"
  >
    <a
      href="/genabackis"
      onClick={[Function]}
    >
      <img
        alt="Genabackis Map Thumbnail"
        className="map-button"
        src="continent-icons/genabackis-icon.svg"
      />
    </a>
    <a
      href="/seven-cities"
      onClick={[Function]}
    >
      <img
        alt="Seven Cities Map Thumbnail"
        className="map-button"
        src="continent-icons/seven-cities-icon.svg"
      />
    </a>
    <a
      href="/quon-tali"
      onClick={[Function]}
    >
      <img
        alt="Quon Tali Map Thumbnail"
        className="map-button"
        src="continent-icons/quon-tali-icon.svg"
      />
    </a>
  </div>,
  <div />,
]
`;
